source/uwp/AdaptiveCardsNuget/AdaptiveCards.UWP.Beta.nuspec

```diff
@@ -13,22 +13,22 @@
         <tags>Adaptive Cards</tags>
     </metadata>
     <files>
-        <file src="..\$Configuration$\XamlCardRenderer\AdaptiveCards.XamlCardRenderer.winmd" target="lib\uap10.0"/>
+        <file src="..\$Configuration$\AdaptiveCardRenderer\AdaptiveCards.Uwp.winmd" target="lib\uap10.0"/>
 
-        <file src="..\$Configuration$\XamlCardRenderer\AdaptiveCards.XamlCardRenderer.dll" target="runtimes\win10-x86\native"/>
-        <file src="..\$Configuration$\XamlCardRenderer\AdaptiveCards.XamlCardRenderer.lib" target="runtimes\win10-x86\native"/>
-        <file src="..\$Configuration$\XamlCardRenderer\AdaptiveCards.XamlCardRenderer.pri" target="runtimes\win10-x86\native"/>
-        <file src="..\$Configuration$\XamlCardRenderer\AdaptiveCards.XamlCardRenderer.pdb" target="runtimes\win10-x86\native"/>
+        <file src="..\$Configuration$\AdaptiveCardRenderer\AdaptiveCards.Uwp.dll" target="runtimes\win10-x86\native"/>
```

Comment:
What is this AdaptiveCardRenderer folder? I'm not sure this will pack correctly, unless thats a custom out dir in the project?

Reply:
The Renderer project folder was renamed from XamlCardRenderer to AdaptiveCardRenderer. The build outputs the artifacts to that folder.